sql-plugin/src/main/scala/org/apache/spark/sql/rapids/GpuOrcFileFormat.scala

```diff
@@ -44,7 +44,13 @@ object GpuOrcFileFormat extends Logging {
 
   def tagGpuSupport(meta: RapidsMeta[_, _, _],
                     spark: SparkSession,
-                    options: Map[String, String]): Option[GpuOrcFileFormat] = {
+                    options: Map[String, String],
+                    schema: StructType): Option[GpuOrcFileFormat] = {
+
+    val unSupportedTypes = schema.filter(field => !GpuOverrides.isSupportedType(field.dataType))
```

Comment:
looks the same as in GpuParquetFileFormat.scala, consider a shared utils.

Reply:
Its not exactly the same. The predicate being tested is different. If you still think we can benefit from refactoring this I can do it

Reply:
the difference is parameterizable but it's not that big of a deal given it's just a few lines, up to you